__docs__/wonder-blocks-typography/body-text.stories.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from "react";
+import {Meta, StoryObj} from "@storybook/react";
+import {StyleSheet} from "aphrodite";
+import packageConfig from "../../packages/wonder-blocks-typography/package.json";
+
+import ComponentInfo from "../components/component-info";
+import TypographyArgTypes from "./typography.argtypes";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+import {BodyText} from "@khanacademy/wonder-blocks-typography";
+import {spacing} from "@khanacademy/wonder-blocks-tokens";
+
+/**
+New component for Thunderblocks body text.
```

Comment:
For my own understanding, should teams use the `BodyText` and `Heading` components in the classic experience? 

Also, how do teams know when they should use the `BodyText` or `Heading` components? Is it something they can inspect in Figma? Maybe we can document something around this so teams know to use these components instead of re-implementing the typography styles!

Reply:
To be honest, I don't know on the classic experience part. But we could chat about it and try to cover it in this docs issue: https://khanacademy.atlassian.net/browse/WB-1955

Reply:
The short answer is yes, teams should use these `BodyText` and `Heading` components in the classic experience.

They are going to use them as soon as we start integrating these components in the `Classroom` experience as there are some UI components that are shared between Classic and Classroom (the ones that live inside the `static/libs` folder.

In general, our strategy will be to create new components having in mind that these should always fallback to the classic experience.

Reply:
Related to our longer discussion on Slack, we'll have to map the theme for these components for the Classic experience. TB uses a different size and line-height scale (Heading and Body styles, vs. global font tokens). I pushed an update to add component-level theming for these components, so there is a space to add the classic values. But they'll have to be determined first!

Reply:
Classic sizing has been added to this PR, so `BodyText` and `Heading` can be used outside of the Classroom theme!